packages/frontend/src/components/ExpandableContainer.tsx

```diff
@@ -11,28 +11,39 @@ interface ExpandableContainerProps {
 
 export function ExpandableContainer(props: ExpandableContainerProps) {
   return (
-    <div className="ExpandableContainer">
```

Comment:
I thought we got rid of those classes in previous PR?

Reply:
Nope, these are the classes responsible for being queryable in scripts. Previously we deleted Sass classes